.github/workflows/tuist.yml

```diff
@@ -60,10 +60,10 @@ jobs:
           ./fourier test tuist unit
   release_build:
     name: Release build with Xcode ${{ matrix.xcode }}
-    runs-on: macOS-latest
+    runs-on: macOS-11
     strategy:
       matrix:
-        xcode: ['12.2', '12.4'] # keep 12.2 to ensure backwards compatibility
+        xcode: ['12.4', '12.5.1', '13.0'] # keep 12.2 to ensure backwards compatibility
```

Comment:
@fortmarek @pepibumur @kwridan Do we need to keep 12.2 compatibility? if yes, we should setup another workflow to run on the 10.15 executor

Reply:
Considering 13.0 is right around the corner, I think it should be fine to sunset 12.2